### PR TITLE
0.2/734 new style register page

### PR DESCRIPTION
**Pull request by alexalala**
https://trello.com/c/wbUFgg0G/734-new-style-register-page

Trying to prune and simplify styles as well

🐸

### CODE REVIEW

File: public/css/main.css [hunk 1]
```diff
@@ -339,41 +316,50 @@ strong {
...
+.col {
+	width: 100%;
+	height: 75vh;
+	background: #EEEEEE;
+	margin: 1rem;
 }
 
-.col {
+.col > div:first-child {
+	background-color: #89d33c;
```
Comment (charlesstrange2) [inline public/css/main.css]: That green is quite sharp. The G in the logo is a murkier #A6CA47.

**Review by charlesstrange2 — APPROVED** [review]
Wow! that's really green on my monitor!
Looks lovely otherwise.

File: public/css/main.css (hunk 1 above)
Comment (alexalala) [reply public/css/main.css]: Yeah I thought so, the palleton said the green we've got works better though, will give the other one a try :D

**Comment by charlesstrange2** [comment]
It felt a bit "high energy"; :-/

**Comment by charlesstrange2** [comment]
Approved, :shipit:

**Pull request merged**